latitudesh/lsh
# Feat: Add support to interactive mode to the servers actions

## What

This PR adds the interactive mode to the servers actions: create, update and destroy. It also introduces a few changes to the operations, aiming to bring more readability and make changes easier to implement.

As a tradeoff, we are deviating from the generated code standards, but as we already made many changes to the auto-generated files, this shouldn't be a big issue.

## How to test

- Run the command `lsh servers create` passing all the required flags
- [ ] It should deploy a new server to your account
- Run the same command without passing any flags
- [ ] It should prompt the fields for creating a server
- [ ] Consider also passing only a few flags to the command, to make sure only the remaining flags will be prompted.

Repeat the tests for:
- [ ] `lsh update`
- [ ] `lsh destroy`
